src/base/utils/fs.cpp

```diff
@@ -355,6 +355,13 @@ bool Utils::Fs::sameFileNames(const QString &first, const QString &second)
 #endif
 }
 
+bool Utils::Fs::isFileInFolder(const QString &file_path)
```

Comment:
Is '/' always a separator in torrent file? May it be for example '\' in Windows or something like that?